nusight2/src/server/nuclearnet/web_socket_proxy_nuclearnet_server.ts

```diff
@@ -219,3 +222,39 @@ class PacketProcessor {
     }
   }
 }
+
+function decodePacketId(event: string, packet: NUClearNetPacket) {
+  const defaultId = 0
+
+  const messageFields = messageFieldsIndex[event]
+  if (messageFields === undefined) {
+    return defaultId
+  }
+
+  const idField = messageFields['id']
+  if (idField === undefined) {
+    return defaultId
+  }
```

Comment:
```suggestion
  const idField = messageFieldsIndex[event].?id
  if (idField === undefined) {
    return 0
  }
```

Reply:
Yep, nice

Reply:
fixed suggestion

Reply:
Just realized you had `.?` instead of `?.` 😅

Reply:
I did not do a typescript 😞